packages/bpk-component-spinner/src/BpkSpinner.js

```diff
@@ -26,18 +28,24 @@ import SPINNER_TYPES from './spinnerTypes';
 
 const getClassName = cssModules(STYLES);
 
-const BpkSpinner = props => {
+type Props = {
+  type: string,
+  className: ?string,
+  alignToButton: boolean,
+};
+
+const BpkSpinner = (props: Props) => {
   const { type, className, alignToButton, ...rest } = props;
 
-  const classNames = ['bpk-spinner', `bpk-spinner--${type}`].map(getClassName);
+  const classNames = [getClassName('bpk-spinner', `bpk-spinner--${type}`)];
```

Comment:
Flow didn't like using `.map(getClassName)` as the `getClassName` function returns mixed types depending on what is passed to it, so flow cannot validate it correctly.

However this is better way we get classNames across our components so cleaning it up to align with others

Reply:
Same comment as before about simplifying this.

Reply:
Did this suggestion and seems to work fine